model/DataQuery.php

```diff
@@ -846,12 +910,13 @@ public function subtract(DataQuery $subtractQuery, $field='ID') {
 	/**
 	 * Select the given fields from the given table.
 	 *
-	 * @param String $table Unquoted table name (will be escaped automatically)
-	 * @param Array $fields Database column names (will be escaped automatically)
+	 * @param string $table Unquoted table name (will be escaped automatically)
+	 * @param array $fields Database column names (will be escaped automatically)
+	 * @return $this
 	 */
 	public function selectFromTable($table, $fields) {
 		$fieldExpressions = array_map(function($item) use($table) {
-			return "\"$table\".\"$item\"";
+			return "\"{$table}\".\"{$item}\"";
```

Comment:
I'm renaming to `sqlColumnForField` and clarifying the need to pass in a class and not a table. It now throws an exception if you pass in a field that doesn't exist on the class, or if you pass in a $class that isn't a class.